Update readme of IE Python samples (#4908)
openvinotoolkit/openvino: inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/README.md

# nGraph Function Creation Python* Sample {#openvino_inference_engine_ie_bridges_python_sample_ngraph_function_creation_sample_README}

> This sample demonstrates how to execute an inference using ngraph::Function to create a network, which uses weights from LeNet classification network.  

Reviewer: ```suggestion
Inference using [nGraph function feature](../../../../../docs/nGraph_DG/build_function.md) to create a LeNet classifications network as an example.
```

                        Optional. Specify the target device to infer on; CPU,
                        GPU, MYRIAD, HDDL or HETERO: is acceptable. The sample
                        will look for a suitable plugin for device specified.
                        Default value is CPU.
  --labels LABELS       Optional. Path to a labels mapping file.
  -nt NUMBER_TOP, --number_top NUMBER_TOP
                        Optional. Number of top results.
```

To run the sample, you need specify a model weights and image:
 - you can use images from the media files collection available at https://storage.openvinotoolkit.org/data/test_data.

> **NOTE**: 
> 
> * This sample supports models with FP32 weights only.
>
> * The `lenet.bin` weights file was generated by the [Model Optimizer](../../../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md) tool from the public LeNet model with the `--input_shape [64,1,28,28]` parameter specified.  
>
> * The original model is available in the [Caffe* repository](https://github.com/BVLC/caffe/tree/master/examples/mnist) on GitHub\*.

You can do inference of an image using a pre-trained model on a GPU using the following command:

```

> python3 ngraph_function_creation_sample.py -m <path_to_model>/alexnet.xml -i <path_to_image>/car.bmp -d GPU

Reviewer: ```suggestion
python3 ngraph_function_creation_sample.py -m <path_to_model>/lenet.bin -i <path_to_image>/car.bmp -d GPU
```

Reviewer 2: python3 is absent on Windows, why we can't use python?
